.github/workflows/cherry-picks.yml

```diff
@@ -0,0 +1,32 @@
+name: Cherry Pick to remote repository
+on:
+  pull_request_target:
+    types: [closed, labeled]
+permissions:
+  contents: write # so it can comment
+  pull-requests: write # so it can create pull requests
+  actions: write
```

Comment:
A bit curious about this part - according to the https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idpermissions it says this permissions definition is for overriding the permission granted for `GITHUB_TOKEN`, since we're mostly using another token `CHERRY_PICK_TOKEN` here, will this permission setting also take effect for `CHERRY_PICK_TOKEN`?

Reply:
I think you're correct. I tested this in my private repo and it seems to not be necessary when using a dedicated token in both of the actions. The permissions in my repo are not completely the same, so things might differ here but I removed the `permissions` block to give it a try.